builtin/mainmenu/textures.lua

```diff
@@ -145,6 +145,22 @@ function mm_texture.set_game(identifier,gamedetails)
 	
 	local path = gamedetails.path .. DIR_DELIM .."menu" ..
 									 DIR_DELIM .. identifier .. ".png"
+
+	local count = 1
+	for i = 9, 1, -1 do
+		local check_path = path .. "." .. i
+		local file = io.open(check_path,"rb")
+		if file ~= nil then
+			count = i
+			io.close(file)
+			break
+		end
+	end
+	local new_path = path .. "." .. math.random(1, count)
```

Comment:
background.png is used, that is the whole point of the second math.random call.

It is fully backwards compatible